### PR TITLE
UI changes and misc improvements for dataprep

### PR DESCRIPTION
**Pull request by ben-c8y**
Rewrite some UI sections for recently changed UI features. 
Also improve and fix some other text.

### CODE REVIEW

File: content/data-preparation/rule-editor-bundle/test-data.md [hunk 1]
```diff
@@ -17,25 +17,40 @@ Use the **Test data** panel in the [rule editor](/data-preparation/rule-editor/)
...
 The test is now associated with the current rule.
 
+Each input message displays the following properties as chips below the payload editor:
+
+- **Payload type** — the format for displaying and editing the message payload (for example, JSON).
+- **Time** — the timestamp indicating when the transport received the message.
+- **Client ID** — the client (typically a device identifier or device gateway) that sent the message.
+- **Topic** — the MQTT topic the message was published to.
+
+To edit these properties, click the edit icon next to the chips.
```
Comment (BeateRixen) [inline content/data-preparation/rule-editor-bundle/test-data.md]:
```suggestion
To edit these properties, click the edit icon <i class="dlt-c8y-icon-edit1 text-primary icon-20"></i> next to the chips. 
```

Comment (BeateRixen) [reply content/data-preparation/rule-editor-bundle/test-data.md]: Not sure if you use the standard edit icon but usually we add small images in such a case

File: content/data-preparation/rule-editor-bundle/test-data.md [hunk 2]
```diff
@@ -17,25 +17,40 @@ Use the **Test data** panel in the [rule editor](/data-preparation/rule-editor/)
...
 
+We recommend creating additional tests to check for edge cases such as missing values, out of range values, and other inputs that might require special handling in your rule. To do this, open the tests drop-down, and duplicate the existing test to a new one that describes what it's for. You can also generate tests using the AI assistant. For any edge cases where it's not possible to generate a valid {{< product-c8y-iot >}} object, raise a Javascript exception so that an alarm will be created to notify you about the problem. 
+
 1. Click the test name dropdown at the top of the test data panel.
-2. Select **Create new test**.
-3. Enter a name for the test.
-4. Enter a payload in the **Input** field for the new test.
+2. Select an existing test to use as a starting point and click **Duplicate test**. 
+3. Enter a short name for the new test.
+4. Edit the **Input** payload for the case you wish to test, for example missing values, different field types or out of range values.
```
Comment (BeateRixen) [inline content/data-preparation/rule-editor-bundle/test-data.md]:
```suggestion
4. Edit the **Input** payload for the case you wish to test, for example, missing values, different field types, or out-of-range values.
```

File: content/data-preparation/getting-started-dataprep.md [hunk 3]
```diff
@@ -20,62 +20,63 @@ You need:
...
+The wizard has three steps:
 
-1. **Source transport**: select the transport from which messages will be received. MQTT is currently the only supported transport.
-2. **Filter**: specify a topic filter or a client ID filter. These determine which incoming messages this rule processes.
-   - **Topic filter**: Only process messages about specific data types or device functions. For example, `sensors/temperature` captures all temperature readings. Use `*` as a wildcard (e.g., `sensors/factories/France/*` captures all sensor data from that region).
-   - **Client ID filter**: If you do not know the topic for your device data, use the client ID to filter for messages from specific devices or device gateways. For example, your device's serial number or assigned device ID. Use `*` as a wildcard (e.g., `serial-number-1af2`, `gateway-*`).
-   
-   Note that these are not MQTT-style filters --- use `*` rather than `#` or `+`.
-3. **Confirm**: review the source, topic, and client ID, then enter a **name** and **description** for the rule.
+1. **Source** — configure which messages will be handled by this data preparation rule, by specifying the transport and at least one filter:
```
Comment (BeateRixen) [inline content/data-preparation/getting-started-dataprep.md]:
```suggestion
1. **Source** — configure which messages will be handled by this Data Preparation rule by specifying the transport and at least one filter:
```

File: content/data-preparation/getting-started-dataprep.md [hunk 4]
```diff
@@ -20,62 +20,63 @@ You need:
...
+   - **Source transport**: select the transport from which messages are received. Currently this is always MQTT.
+   - **Topic filter**: filter based on the MQTT topic the device messages are received from. Use `*` as a wildcard to match zero or more characters, for example  `/sensors/factories/France/*/temperature/*` captures all temperature sensor data from that region. Note that transport-specific wildcards such as `+` and `#` are not supported here.
```
Comment (BeateRixen) [inline content/data-preparation/getting-started-dataprep.md]:
```suggestion
   - **Topic filter**: filter based on the MQTT topic the device messages are received from. Use `*` as a wildcard to match zero or more characters, for example,  `/sensors/factories/France/*/temperature/*` captures all temperature sensor data from that region. Note that transport-specific wildcards such as `+` and `#` are not supported here. 
```

File: content/data-preparation/getting-started-dataprep.md [hunk 5]
```diff
@@ -20,62 +20,63 @@ You need:
...
-   
-   Note that these are not MQTT-style filters --- use `*` rather than `#` or `+`.
-3. **Confirm**: review the source, topic, and client ID, then enter a **name** and **description** for the rule.
+1. **Source** — configure which messages will be handled by this data preparation rule, by specifying the transport and at least one filter:
+   - **Source transport**: select the transport from which messages are received. Currently this is always MQTT.
+   - **Topic filter**: filter based on the MQTT topic the device messages are received from. Use `*` as a wildcard to match zero or more characters, for example  `/sensors/factories/France/*/temperature/*` captures all temperature sensor data from that region. Note that transport-specific wildcards such as `+` and `#` are not supported here. 
+   - **Client ID filter**: if you do not know the topic, filter by the client ID of your device or gateway instead. Use `*` as a wildcard if needed, for example `serialnumber-1aaaf2ac-*`, `device-gateway-1234*`.
```
Comment (BeateRixen) [inline content/data-preparation/getting-started-dataprep.md]:
```suggestion
   - **Client ID filter**: if you do not know the topic, filter by the client ID of your device or gateway instead. Use `*` as a wildcard if needed, for example, `serialnumber-1aaaf2ac-*`, `device-gateway-1234*`.
```

File: content/data-preparation/getting-started-dataprep.md [hunk 6]
```diff
@@ -20,62 +20,63 @@ You need:
...
+   If your device is already connected to Cumulocity you can now capture a device message to test with using the source data you entered. Alternatively you can skip the live capture and enter your device messages for testing manually after creating the rule.
```
Comment (BeateRixen) [inline content/data-preparation/getting-started-dataprep.md]:
```suggestion
   If your device is already connected to {{< product-c8y-iot >}}, you can now capture a device message to test with using the source data you entered. Alternatively, you can skip the live capture and enter your device messages for testing manually after creating the rule. 
```

File: content/data-preparation/rule-creation-management.md [hunk 7]
```diff
@@ -6,27 +6,51 @@ sector:
...
+
+To use live capture to select a sample message for your new rule:
+- Click **Create rule** to open the rule creation wizard. 
+- Use the **Source** fields on the first page to specify which messages should be captured for the rule you are creating. You can use `*` wildcards if you don't know the exact topic or client ID, but make sure your filters are specific enough to focus on the device you wish to onboard. 
+- Click **Start live capture from device**. The wizard advances to the live capture step and begins listening for messages that match your filters. Incoming messages appear in a list as they arrive. Each entry shows the timestamp, topic, and client ID.
+- Click a message in the list to select it. The selected message is added as a test message when the rule is created.
+- If you do not see the expected messages or realize you have entered the wrong filters, click **Edit Capture Source** to change the topic or client ID filter and start a new capture session. Alternatively, you can click **Back** to cancel live capture and click **Enter test data later** to proceed directly to the confirm step.
```
Comment (BeateRixen) [inline content/data-preparation/rule-creation-management.md]: **Edit Capture Source** > should actually be **Edit capture source**. Not sure if this needs to be adapted in the UI as well.

File: content/data-preparation/rule-creation-management.md [hunk 8]
```diff
@@ -62,13 +85,9 @@ Click the **Reload** button to refresh the rules list and show the latest deploy
...
 In the rule editor you can update the smart function code, add test data, and run tests. For a full reference of the rule editor and its panels, see [Rule editor](/data-preparation/rule-editor/).
 
-### To duplicate a rule {#duplicate-rule}
-
-- Click the **Duplicate** icon next to the rule in the list. A copy of the rule is created in the **Draft** state. The duplicate has the same smart function, test data, topic filter, and client ID filter as the original.
-
 ### To delete a rule {#delete-rule}
 
-- Click the **Delete** icon next to the rule in the list. The rule is permanently deleted. This action cannot be undone.
+- Click the **Delete** icon next to the rule in the list. The rule is permanently deleted from both the draft and deployed state. This action cannot be undone.
```
Comment (BeateRixen) [inline content/data-preparation/rule-creation-management.md]:
```suggestion
- Click the **Delete** icon <i class="dlt-c8y-icon-editing-trash text-danger icon-20"></i> next to the rule in the list. The rule is permanently deleted from both the draft and deployed state. This action cannot be undone.
```

**Review by BeateRixen — APPROVED** [review]

File: content/data-preparation/rule-creation-management.md (hunk 7 above)
Comment (ben-c8y) [reply content/data-preparation/rule-creation-management.md]: yes, have fixed in UI just now and updated the doc
